spk/tvheadend/src/service-setup.sh

```diff
@@ -6,7 +6,7 @@ HTTPP=9981
 HTSPP=9982
 
 # Replace generic service startup, run service in background
-GRPN=`id -gn ${EFF_USER}`
+GRPN=$(id -gn ${EFF_USER})
 HOME_DIR="${SYNOPKG_PKGDEST}/var"
```

Comment:
Looking further at the code without mentioning the backup functionality that needs reworking eventually
(as inherited from original package and not working so much)

Isn't there a `${SYNOPKG_PKGVAR}` variable?